c7n/tags.py

```diff
@@ -275,22 +291,23 @@ def __call__(self, i):
         if ':' not in v or '@' not in v:
             return False
 
-        msg, tgt = v.rsplit(':', 1)
+        msg, tgt = v.split(':', 1)
```

Comment:
this should be rsplit, split imposes additional constraints on message templates for tag values.